perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -362,6 +374,10 @@ sub inspect : Private {
         }
 
         if ($valid) {
+            if ( $reputation_change != 0 && !$problem->get_extra_metadata('user_reputation_set') ) {
+                $problem->user->update_reputation($reputation_change);
+                $problem->set_extra_metadata( user_reputation_set => 1)
```

Comment:
No, I just meant do you need to bother setting/checking the user_reputation_set metadata, as it seems very unlikely that a report could have this happen twice anyway.